cfme/tests/cloud/test_providers.py

```diff
@@ -320,15 +320,17 @@ def test_openstack_provider_has_api_version():
 class TestProvidersRESTAPI(object):
     @pytest.yield_fixture(scope="function")
     def arbitration_profiles(self, rest_api, setup_a_provider):
+        num_profiles = 2
         provider = rest_api.collections.providers.get(name=setup_a_provider.name)
         body = []
         providers = [{'id': provider.id}, {'href': provider.href}]
-        for i in range(2):
+        for i in range(num_profiles):
             body.append({
                 'name': 'test_settings_{}'.format(fauxfactory.gen_alphanumeric(5)),
                 'provider': providers[i % 2]
             })
         response = rest_api.collections.arbitration_profiles.action.create(*body)
+        assert len(response) == num_profiles
```

Comment:
**Optional**
Consider a soft_assert here, looks like the tests don't have a hard dependency on the number of profiles given by the fixture

This ties in with my comments below. If this is a soft_assert it could create ambiguity in the tests as to whether or not the test executed against 0, 1, or `num_profiles` profiles.